src/UI/Implementation/Component/Modal/Renderer.php

```diff
@@ -47,27 +47,27 @@ public function registerResources(ResourceRegistry $registry) {
 	 * @param Component\Modal\Modal $modal
 	 * @param string $id
 	 */
-	protected function registerSignals(Component\Modal\Modal $modal, $id) {
+	protected function registerSignals(Component\Modal\Modal $modal) {
 		$show = $modal->getShowSignal();
 		$close = $modal->getCloseSignal();
-		$js = $this->getJavascriptBinding();
```

Comment:
[There](https://github.com/ILIAS-eLearning/ILIAS/pull/578/files#diff-7940fff29529531d00fa71307b4fa42bR57) you go.